addons/survey/controllers/main.py

```diff
@@ -151,6 +151,8 @@ def _redirect_with_error(self, access_data, error_key):
             return request.render("survey.survey_auth_required", {'survey': survey_sudo, 'redirect_url': redirect_url})
```

Comment:
In the commit message, can you write: "Task-2960271"

And not "task- 2960271" (no spacing).

It's important because the extra spacing will make github try to tie the number (2960271) to a commit hash, which can give strange results.